### ISSUE TITLE
predict_classes 

### ISSUE THREAD

**Issue by PARKER-X** [krishnaik06/Fake-New-LSTM#2]
y_pred=model.predict_classes(X_test)
then i use predict only instead of predict classes
but accuracy and matrix shows an error
then i use argmax after predict it gives less accuracy 

how to get 90+accuracy??

y_pred = model1.predict(x_test)
y_pred[1]
class_x=np.argmax(y_pred,axis=1)
class_x
array([0, 0, 0, ..., 0, 0, 0])

from sklearn.metrics import accuracy_score, confusion_matrix
score = accuracy_score(y_test, class_x)
score
0.5578342904019689

**Comment by 12194916** [comment]
While giving to confusion_matrix and accuracy_score, y_pred needs to be rounded. "confusion_matrix(y_test,y_pred.round( ))". It works.
